streams/src/main/java/org/apache/kafka/streams/kstream/Transformer.java

```diff
@@ -43,7 +43,6 @@
  * @param <R> {@link KeyValue} return type (both key and value type can be set
  *            arbitrarily)
  * @see TransformerSupplier
- * @see KStream#transform(TransformerSupplier, String...)
```

Comment:
We will also remove `Transformer` in a follow up PR completely, right? So no need to have any updates in this PR?

Reply:
Although we remove it, I think it helpful to update it to prevent `gradle javadoc` to fail. I am checking if the modifications are right this way. Is there any problem if I go this way? Or some suggestion?